<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Temperatura:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Convertir a:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="centi_kelvin">
         <property name="text">
          <string>Centigrados a Kelvin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="kelvin_centi">
         <property name="text">
          <string>Kelvin a Centigrados</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="centi_fahr">
         <property name="text">
          <string>Centigrados a Fahrenheit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="fahr_centi">
         <property name="text">
          <string>Fahrenheit a Centigrados</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_temperatura">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Temperatura:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_calcular">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
